src/test/java/com/meilisearch/integration/SearchTest.java

```diff
@@ -217,13 +183,10 @@ public void testSearchHighlight() throws Exception {
 
 		SearchRequest searchRequest = new SearchRequest("and")
 			.setAttributesToHighlight(new String[]{"overview"});
-		Results res_gson = jsonGson.decode(
-			index.search(searchRequest),
-			Results.class
-		);
-		assertEquals(20, res_gson.hits.length);
-		assertTrue(res_gson.hits[0].getFormatted().getOverview().contains("<em>"));
-		assertTrue(res_gson.hits[0].getFormatted().getOverview().contains("</em>"));
+
+		SearchResult searchResult = index.search(searchRequest);
+
+		assertEquals(20, searchResult.getHits().size());
```

Comment:
@curquiza @eskombro 

Currently I've implemented `hits` as `Array<HashMap<String, Object>>`, but not sure whether this is the best way.
Maybe it could be better to use `Array<Object>` to allow end user to serialize the format inside array more easily...

How do you think?